packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/alb/application-load-balancer.ts

```diff
@@ -65,6 +65,69 @@ export interface ApplicationLoadBalancerProps extends BaseLoadBalancerProps {
    * @default - Duration.seconds(3600)
    */
   readonly clientKeepAlive?: Duration;
+
+  /**
+   * Indicates whether the Application Load Balancer should preserve the Host header in the HTTP request
+   * and send it to the target without any change.
+   *
+   * @default false
+   */
+  readonly preserveHostHeader?: boolean;
+
+  /**
+   * Indicates whether the two headers (x-amzn-tls-version and x-amzn-tls-cipher-suite),
+   * which contain information about the negotiated TLS version and cipher suite,
+   * are added to the client request before sending it to the target.
+   *
+   * The x-amzn-tls-version header has information about the TLS protocol version negotiated with the client,
+   * and the x-amzn-tls-cipher-suite header has information about the cipher suite negotiated with the client.
+   *
+   * Both headers are in OpenSSL format.
+   *
+   * @default false
+   */
+  readonly xAmznTlsVersionAndCipherSuiteHeaders?: boolean;
```

Comment:
can you add `@see` values with links to AWS docs?

Reply:
@badmintoncryer Since the urls are the same, I think it'd make more sense to add the `@see` one time under `Properties for defining an Application Load Balancer` ?

Reply:
@msambol I was unable to retrieve links for each attribute and ended up entering the same URL...
I made the revisions just as you suggested!